src/key-value.js

```diff
@@ -10,19 +10,29 @@ export default React.createClass({
       valueItem: PropTypes.string
     })),
     onChange: PropTypes.func,
-    customAddButtonRenderer: PropTypes.func
+    customAddButtonRenderer: PropTypes.func,
+    keyInputPlaceholder: PropTypes.string,
+    valueInputPlaceholder: PropTypes.string,
+    hideKeyInputLabel: PropTypes.bool,
+    hideValueInputLabel: PropTypes.bool
   },
   getDefaultProps() {
     return {
       rows: [],
-      onChange: () => {}
+      onChange: () => {},
+      keyInputPlaceholder: 'Enter the meta key\'s title',
```

Comment:
the default value of the placeholder should be an empty string b/c the user might not want to have them